editor/inspector/contributions/node.js

```diff
@@ -676,6 +676,11 @@ const Elements = {
                             break;
                         }
                         case 'reset': {
+                            // restore-prefab 内置undo;
+                            if (undoID) {
+                                await endRecording(undoID, true);
+                                undoID = undefined;
```

Comment:
There seems to be some issues here. If the "reset" event is triggered, the target node is set to "undefined." If other events, such as "change," are triggered afterward, will they no longer be responsive?

Reply:
undo is record for reset operation here, so remove the code.